src/ape/api/accounts.py

```diff
@@ -135,7 +165,21 @@ def transfer(
 
         return self.call(txn, send_everything=value is None)
 
-    def deploy(self, contract: "ContractContainer", *args, **kwargs) -> "ContractInstance":
+    def deploy(self, contract: ContractContainer, *args, **kwargs) -> ContractInstance:
+        """
+        Create a smart contract on the blockchain.
+
+        Method Limitations:
+            Smart Contract must compile before deploying.
+            A provider must active.
+
+        Args:
+            contract (:class:`~ape.api.contracts.ContractContainer`):
```

Comment:
Ah these links have changed

I believe it is `~ape.contracts.ContractContainer` now (just take out the `.api`)